src/utils/topic.ts

```diff
@@ -31,7 +31,11 @@ export const buildUserInviteTopic = (walletAddr: string): string => {
   // EIP55 normalize the address case.
   return buildContentTopic(`invite-${utils.getAddress(walletAddr)}`)
 }
+
 export const buildUserPrivateStoreTopic = (addrPrefixedKey: string): string => {
   // e.g. "0x1111111111222222222233333333334444444444/key_bundle"
   return buildContentTopic(`privatestore-${addrPrefixedKey}`)
 }
+
+export const buildUserPrivatePreferencesTopic = (identifier: string) =>
+  buildUserPrivateStoreTopic(`${identifier}/allowlist`)
```

Comment:
We should get everyone aligned on what the topic identifier actually is. Pat's doc mentioned the private store topic, but that was before we created these new identifiers.

We should do something new so that this has no chance of interfering with existing private stores (or screwing up our analytics on the growth of the network).

Maybe something like `buildContentTopic(`pppp-${identifier}`)

Reply:
cc @nplasterer

Reply:
So just to be clear it would be `"pppp-$identifier"` not  `"pppp-$identifier/allowlist"`

Reply:
My proposal is that all PPPP events (allow, block, some future feature) would go into one topic, and the client would reduce the events into a consistent state. 

Could be convinced we want to split the topics up. Just more fetching.

Reply:
i think `pppp-$identifier/allowlist` works great and allows for adding more types of preferences in the future.

Reply:
Okay that makes sense @neekolas so in that case it should just be `pppp-$identifier` we make the block and allow list from those events and serve them to them from the class. It just feels confusing to add `allowlist` to the end here when we might have pinned wallets etc... It feels like all the preferences should get loaded at the same time so no need to have separate topics for each type of preference? Unless we think that will restrict us in the future when preference may get more complicated with groups.